default-blog-post.pug

```diff
@@ -15,14 +15,14 @@ head
 	
 	meta(property="og:image"             	content="http://mspudding.com/Images/Duckfriedrice/sizzlingduck.jpg")
 	meta(property="og:image:height" 		content="1919")
-	meta(property="og:image:width" 			gbvb gb content="1536")
+	meta(property="og:image:width" 			content="1536")
 
 	//- <!-- Stylesheeeeets! -->
```

Comment:
Just use `// Stylesheeets!` and it will make a nice html comment for you
only use `//-` if you want to hide the comment from output which is not neccesary here.